go.mod

```diff
@@ -93,64 +96,58 @@ require (
 	github.com/google/uuid v1.6.0 // indirect
 	github.com/gorilla/websocket v1.5.3 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.20.0 // indirect
-	github.com/hashicorp/hcl v1.0.0 // indirect
```

Comment:
Viper got rid of hcl, so now we can get rid of hcl (hcl was not great from a licensing perspective): https://github.com/kubernetes-sigs/cluster-api/issues/9181

Reply:
Finally :+1: